app/controllers/concerns/accessible.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+module Accessible
+  extend ActiveSupport::Concern
+
+  def execute_if_accessible(subject: nil, redirect: true)
+    return unless subject
```

Comment:
Maybe let's make `subject` just a plain parameter (not-kwarg, no default value...)
It will make this line unnecessary, and also will allow you to get rid of the repeating `subject: ` everywhere in the calls.